### ISSUE TITLE
Ambiguous name _≡⟨_⟩_ on Data/Digit.agda

### ISSUE THREAD

**Issue by asr** [agda/agda-stdlib#276]
I got the following error on master:

```
$ agda --version
Agda version 2.5.3
```

```
$ make test
...
<SOME_PATH>/src/Data/Digit.agda:100,5-106,8
Ambiguous name _≡⟨_⟩_. It could refer to any one of
  .Data.Nat.Properties.≤-Reasoning._≡⟨_⟩_
  bound at
  <SOME_PATH>/src/Relation/Binary/PreorderReasoning.agda:54,1-7
  .Data.Nat.Properties.≤-Reasoning._≈⟨_⟩_
  bound at
  <SOME_PATH>/src/Relation/Binary/PreorderReasoning.agda:51,1-7
(hint: Use C-c C-w (in Emacs) if you want to know why)
when scope checking
2 + x + (x + (1 + x) * k + r) ≡⟨
solve 3
(λ x r k →
   con 2 :+ x :+ (x :+ (con 1 :+ x) :* k :+ r) := r :+ (con 1 :+ x) :*
   (con 2 :+ k))
refl x r k
⟩ r + (1 + x) * (2 + k) ∎
```

**Comment by MatthewDaggitt** [comment]
Sorry, caused by ab9451a. That was very lax of me. Fixing it now.

**Comment by MatthewDaggitt** [comment]
Fixed with 630311b
